clang/test/SemaSYCL/unnamed-kernel.cpp

```diff
@@ -79,8 +90,9 @@ struct MyWrapper {
 
     using InvalidAlias = InvalidKernelName4;
 #ifndef __SYCL_UNNAMED_LAMBDA__
-    // expected-error@Inputs/sycl.hpp:220 {{kernel needs to have a globally-visible name}}
-    // expected-note@23 {{InvalidKernelName4 declared here}}
+    // expected-error@Inputs/sycl.hpp:220 {{invalid type 'MyWrapper::InvalidKernelName4' used in kernel name. Kernel name should be globally-visible}}
+    // expected-note@Inputs/sycl.hpp:220 {{Invalid kernel name is 'MyWrapper::InvalidKernelName4'}}
+    // expected-note@29 {{InvalidKernelName4 declared here}}
```

Comment:
note: `InvalidKernelName4 declared here` seems a but redundant when we already have a error which specifies full kernel name.

Reply:
I think this is redundant when we print the type in error message. I believe pointing out the problematic type (whether in error message itself or using this note) is useful even if we provide the full kernel name. I can remove this note. The only reason I kept it is because it points to the actual location (as opposed to headers) and I was wondering if that is useful information to keep

Reply:
Removed note